Source/FAQView.swift

```diff
@@ -360,28 +392,48 @@ class FAQViewCell: UITableViewCell {
   func configure(configuration: FAQConfiguration) {
     self.backgroundColor = configuration.cellBackgroundColor
     self.questionLabel.textColor = configuration.questionTextColor
-    self.answerLabel.textColor = configuration.answerTextColor
+    self.answerTextView.textColor = configuration.answerTextColor
     self.questionLabel.font = configuration.questionTextFont
-    self.answerLabel.font = configuration.answerTextFont
+    self.answerTextView.font = configuration.answerTextFont
+    if let dataDetectorTypes = configuration.dataDetectorTypes {
+      self.answerTextView.dataDetectorTypes = dataDetectorTypes
+    }
+    if let tintColor = configuration.tintColor {
+      self.answerTextView.tintColor = tintColor
+    }
   }
   
   func expand(withAnswer answer: String, animated: Bool) {
-    self.answerLabel.text = answer
+    answerTextView.text = answer
+    answerTextView.isHidden = false
     if animated {
-      self.answerLabel.alpha = 0
+      answerTextView.alpha = 0
       UIView.animate(withDuration: 0.5, animations: {
-        self.answerLabel.alpha = 1
+        self.answerTextView.alpha = 1
       })
     }
-    self.answerLabelBottom.constant = 20
-    self.update(arrow: .Up, animated: animated)
+    answerTextViewBottom.constant = 20
+    update(arrow: .Up, animated: animated)
   }
   
+  func expand(withAttributedAnswer attributedAnswer: NSAttributedString, animated: Bool) {
```

Comment:
It looks like I must have messed up when I tried to resolve conflicts. (As I stated before, I am not proficient with Github yet.) Let me examine my code a bit. My fork works correctly, but I must not have all the changes you made previously.